### ISSUE TITLE
EfieldToVoltageConverter does not work for far away channels

### ISSUE THREAD

**Issue by christophwelling** [nu-radio/NuRadioReco#159]
If a channel is too far away from the station center, the efieldToVoltageConverter does not work any more. When the starting bin for the signal in the channel trace is calculated, the result becomes negative, which causes an error.
This can be avoided by increasing the pre_pulse_time, but ideally this should work automatically.

**Comment by cg-laser** [comment]
can you be more specific of the error? Stacktrace? I don't see why this would be a problem in general, probably the code needs to be made more robust.

**Comment by anelles** [comment]
I would mark this as a science problem, rather than a computational problem. We have to develop algorithms that work with extended sources. The current code was never intended to work for more than one surface station.

**Comment by christophwelling** [comment]
No, this is a computing problem. The efieldToVoltageConverter should make the trace long enough to fit the pulse automatically, but for some reason it can happen that in line 182 it tries to use a negative start_bin, which throws an error. So there is something wrong with how the trace length is calculated vs. how that start_bin is calculated.
Practically, this seems to only happen for very large distances between the channel and the station center, so it's pretty low priority. I just made this issue to keep track.

**Comment by anelles** [comment]
Also here, should we just document this issue or do something about it? Especially if it is only low priority as elaborated above.
